go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/authzed/zed
 
-go 1.23.8
+go 1.24
```

Comment:
Should this be in a separate pull request?

Reply:
no because I needed it for `t.Context()`